### ISSUE TITLE
Resize font and window together

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#233]
```
It might be useful to provide a way to resize font and window together, such 
that the number of rows and columns stays approximately constant. (It can only 
be approximate because font sizes are not as flexible as window sizes.)

This should apply both to font resizes with Ctrl+plus/minus or Ctrl+mousewheel, 
and to window resizes e.g. by dragging the window edges or switching between 
normal/maximised/fullscreen states.

PuTTY has a config dialog option for this (along with generally somewhat flaky 
resize handling). I think it would be better though to be able to activate this 
behaviour only when needed. Holding Shift while performing one of the relevant 
actions seems an appropriate way to do this, so for example pressing 
Shift+Alt+F11 would zoom to fullscreen.

See also issue 79 for ideas on how to handle the problem of the window going 
off screen when increasing font size. Issue 204 was about the fullscreen aspect 
of this only.

Without this enhancement, users simply have to resize font and window 
separately.
```

Original issue reported on code.google.com by `andy.koppe` on 6 Nov 2010 at 6:59

**Comment by mintty** [comment]
Implemented optional sync of window scaling with font scaling, triggered by Shift key:
•	Shift+Ctrl+keypad-Plus
•	Shift+Ctrl+keypad-Minus
•	Shift+Ctrl+mouse-wheel
•	ESC sequence OSC 7777
See also #204.

**Comment by mintty** [comment]
Released search bar related tweaks in 2.6.2.

**Comment by mintty** [comment]
Released 2.7.8 which does not capture Ctrl+Shift+0 anymore if it is defined as a Windows shortcut.

**Comment by mintty** [comment]
General rule is now: when holding Shift while resizing or zooming, window size and font size are kept in sync.
